### PR TITLE
NMA-6773: Change light mode disabled colors to new Black 55 primitive

### PR DESCRIPTION
**Pull request by sindrenm**

### CODE REVIEW

File: sats-dna/src/main/kotlin/com/sats/dna/colors/SatsLightColors.kt [hunk 1]
```diff
@@ -383,7 +384,7 @@ internal val SatsLightColors = SatsColors(
                 bg = Black5,
                 fg = SatsBlue100,
                 fgAlternate = SatsBlue70,
-                fgDisabled = Black60,
+                fgDisabled = Black55,
```
Comment (SatseHavard) [inline sats-dna/src/main/kotlin/com/sats/dna/colors/SatsLightColors.kt:387]: "On Surface" -> "Secondary/Disabled" is still "Black 60" on Figma. Does it need to be updated on Figma?

**Review by SatseHavard — APPROVED** [review]
Gucci

File: sats-dna/src/main/kotlin/com/sats/dna/colors/SatsLightColors.kt (hunk 1 above)
Comment (sindrenm) [reply sats-dna/src/main/kotlin/com/sats/dna/colors/SatsLightColors.kt:387]: [Asked about this on Slack.](https://satsdigital.slack.com/archives/CQU1TJ47Q/p1716803763399509?thread_ts=1716458440.769699&cid=CQU1TJ47Q) It doesn't seem right that it's not changed alongside the others. 🤔

Comment (sindrenm) [reply sats-dna/src/main/kotlin/com/sats/dna/colors/SatsLightColors.kt]: Yeah, turns out that was actually on purpose. Nice catch! Reverted. 👍

**Pull request merged**